src/widgets/helper/NotebookTab.hpp

```diff
@@ -60,6 +61,8 @@ class NotebookTab : public Button
     bool isLive() const;
 
     void setHighlightState(HighlightState style);
```

Comment:
This function is only used once (and in tests). It should probably be renamed to avoid confusion.

Reply:
yeah its used when a tab is duplicated

Reply:
// needs to be checked, with the duplicate tab